core/indigo-core/molecule/src/molecule_cdxml_loader.cpp

```diff
@@ -111,8 +133,14 @@ void MoleculeCdxmlLoader::_parseCollections(BaseMolecule& mol)
         case kCDXNodeType_ElementList:
             atoms.push_back(node_idx);
             break;
-        case kCDXNodeType_ExternalConnectionPoint:
-            break;
+        case kCDXNodeType_ExternalConnectionPoint: {
+
+            auto& fn = nodes[_fragment_nodes.back()];
+            if (fn.connections.size() == 0)
+                fn.ext_connections.push_back(node.id);
+        }
+        break;
+        case kCDXNodeType_Nickname:
```

Comment:
is it ok that case without break?

Reply:
break statement can be placed inside brackets of 'case' or outside. This is a question of code style. Both variants work with the same manner.